src/App.php

```diff
@@ -282,7 +291,7 @@ public function url($args = [])
         $page = $args[0];
         unset($args[0]);
 
-        $url = $page ? ($page.'.php') : '';
+        $url = $page ? ($page.($extension ? '.'.$extension : '')) : '';
```

Comment:
i want to encourage integrators (people who integrate Agile UI into a framework) to rewrite url() method with the goal to properly create links between PAGES. Passing extension to a page is not right.